pkg/internal/testhelpers/postgres_container.go

```diff
@@ -294,13 +303,16 @@ func StartPGContainer(
 	printLogs bool,
 ) (testcontainers.Container, io.Closer, error) {
 	var image string
-	PGMajor := "13"
+	PGMajor := "14"
+	if extensionState.UsesPG13() {
+		PGMajor = "13"
+	}
 	if extensionState.UsesPG12() {
```

Comment:
Nit: using `else if` here seems more appropriate.

Reply:
Why not put this logic into `extensionState` and add a method called `GetPGMajorVersion()`? PG15 will happen and this part will just grow.

Reply:
Looking at it a bit more... Image used is determined by extension state, this whole logic could be moved in there.

Would nicely encapsulate things for when we update this in the future.

Reply:
This would also allow us to write tests to check the logic without spinning up containers.

Reply:
Yep agreed. Changed